package.json

```diff
@@ -146,6 +146,8 @@
     "@types/sinon": "4.1.3",
     "@types/sinon-chai": "2.7.29",
     "@types/source-map": "^0.5.2",
+    "@typescript-eslint/eslint-plugin": "^4.29.1",
+    "@typescript-eslint/parser": "^4.29.1",
```

Comment:
I feel like these are welcome changes... but are they necessary in this PR?

Reply:
I can spliot actually, it was due to I got annoyed editor actually failed to initiate eslint. Up to you - but this should be harmless for the fix itself & other editor experiences.